state-network.md

```diff
@@ -175,157 +175,11 @@ A node is said to be *"interested"* in a piece of content if `distance(node_id,
-    * It is invalid to return multiple ENR records for the same `node_id`.
-
-> Note: If the number of ENR records cannot be encoded into a single message, then they should be sent back using multiple messages, with the `total` field representing the total number of messages that are being sent.
-
-#### Find Content (0x05)
-
-Request either the data payload for a specific piece of content on the network, **or** ENR records of nodes that are closer to the requested content.
-
-```
-message_id := 0x05
-type       := request
-sedes      := Container(content_key: byte_list)
-```
-
-* `content_key` the pre-image key for the content being requested..
-
-
-#### Found Content (0x06)
-
-Response message to Find Content (0x05).
-
-This message can contain **either** the data payload for the requested content *or* a list of ENR records that are closer to the content than the responding node.
-
-```
-message_id := 0x06
-type       := response
-sedes      := Container(enrs: List[byte_list, max_length=32], payload: byte_list)
-```
-
-* `enrs`: List of bytestrings, each of which is an RLP encoded ENR record.
-    * Individual ENR records **MUST** be closer to the requested content than the responding node.
-    * It is invalid to return multiple ENR records for the same `node_id`.
-    * This field **must** be empty if `payload` is non-empty.
-* `payload`: bytestring of the requested content.
-    * This field **must** be empty if `enrs` is non-empty.
-    
-> A response with an empty `payload` and empty `enrs` indicates that the node is not aware of any closer nodes, *nor* does the node have the requested content.
-
-
-#### Offer (0x07)
-
-Offer a set of content keys that this node has proofs available for.
-
-```
-message_id := 0x07
-type       := request
-sedes      := Container(content_keys: List[byte_list, max_length=64])
-```
-
-The payload of this message is a list of encoded `content_key` entries.
-
-
-#### Accept (0x08)
-
-Response message to Offer (0x07).
-
-Signals interest in receiving the offered data fro the corresponding Offer message.
-
-
-```
-message_id := 8
-type       := response
-sedes      := Container(connection_id: bytes4, content_keys: BitList[max_length=64]]
-```
-
-* `connection_id`: ConnectionID to be used for a uTP stream
-    * ConnectionID values should be randomly generated.
-* `content_keys`: Signals which content keys are desired
-    * A bit-list corresponding to the offered keys with the bits in the positions of the desired keys set to `1`.
-
-Upon *sending* this message, the requesting node should *listen* for an incoming uTP stream with the generated `connection_id`.
-
-Upon *receiving* this message, the serving node should initiate a uTP stream.
-
-> TODO: how does message framing across the stream work for individual messages.
+The `protocol-id` in the `TALKREQ` message is defined as "portal-state".
 
+All protocol messages are supported.
```

Comment:
These might be better as explicit lists that include all of the messages (so that if we ever add new messages, we don't end up out of sync)